### ISSUE TITLE
the synchronize tick box

### ISSUE THREAD

**Issue by majed17** [Cacti/cacti#2603]
Peace, in the old cacti (before 1.0) there was a tick box to synchronize in the popup real time graph. for example, i would like to resize real time graphs on one monitor while the more important ones one the next monitor are in the biggest size. current behavior is all or nothing :)

**Comment by cigamit** [comment]
Send us a before and after screen shot.

**Comment by majed17** [comment]
sorry, it has been 5 years since i last saw cacti 0.8.8...

**Comment by majed17** [comment]
i am even willing to donate a 25$ for this, it may sound small, but it is huge for a socialist republic!

**Comment by cigamit** [comment]
I think be default, if you use the popout window, they are automatically sync'd.

**Comment by majed17** [comment]
yeah and that is exactly the problem. I don't want them to be synced, hence the need for the option to synchronize!

**Comment by TheWitness** [comment]
I think this is worth considering.
